src/pages/home/HeaderView.js

```diff
@@ -19,6 +19,9 @@ const propTypes = {
     // Decides whether we should show the navigationMenu button
     shouldShowNavigationMenuButton: PropTypes.bool.isRequired,
 
+    // Report ID currently being looked at, use to retrieve more information about the report.
+    reportID: PropTypes.string.isRequired,
```

Comment:
Is there a better way to do this? I passed in the reportID from the `HomePage` since I wasn't sure how else we'd know what report to pull from Onyx...